packages/card/src/css/index.js

```diff
@@ -140,6 +140,11 @@ export default {
     transition: `opacity ${core.motion.speedNormal}`,
     pointerEvents: 'none',
     opacity: 0,
+    zIndex: 10,
+
+    '> *': {
```

Comment:
This is more flexible, for sure.  But it the Card `actionBar` is specifically meant to only allow as children `Card.Action`s.  

Secondarily, it's a much less-performant selector than using `.psds-card__action-bar__button` as a selector (ie, it's `*` vs a className and resolve right to left, so the first-child hierarchy doesn't make it more performant than normal `*`).  But again, this is secondary.  

I'm just wondering... why support this (or look like we support this; "this", being `actionBar` containing `*` kinds of things) if we don't.  

Is there a general design case to be made for supporting things other than `Card.Action` in the `actionBar`?

Reply:
Maybe that's the core of the problems I'm having: card isn't built to support the common use case of including an action-menu in the action bar. This seemed like a more flexible way of doing that rather than asking you to build in support specifically for ActionMenus. If that's the way you'd prefer to do it than that works for me too.

Reply:
It is a good point: that Card should support Card.Action's trigger ActionMenu appearance, because that's in the top 2 most common use cases.  

I guess it's down to a question of: Should Card support ActionMenu appearing *within* its own dom subtree?  Obviously, when making this, I thought no.  But, considering I've heard of people wanting to do it this way a couple times, maybe we go ahead and support it.  React.createPortal still feels cleaner to me, but there is the code of writing the portal rendering vs rendering inline with your Card.Action.

Thinking 2 more mins on this...